apollo-ios/Sources/ApolloAPI/GraphQLOperation.swift

```diff
@@ -121,6 +121,8 @@ public extension GraphQLOperation {
   }
 
   func hash(into hasher: inout Hasher) {
+    hasher.combine(Self.operationType)
```

Comment:
We probably don't need `operationType` since operation names must be unique across all documents. This is something we enforce during codegen validation.